### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by unoebauer**
This PR should improve the general Readme and of the tardisanlysis repo

- [x] add Licence
- [x] add general description of repo to readme
- [x] add example plots

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+* Neither the name of tep nor the names of its
```
Comment (wkerzendorf) [inline LICENSE]: the name of `tep`?

**Comment by wkerzendorf** [comment]
I like the last todo! :+1:

**Comment by unoebauer** [comment]
Done - @wkerzendorf, what do you think? Any comments? Otherwise, I'd merge.

**Comment by wkerzendorf** [comment]
looks very good - merging

**Pull request merged**